forge-gui/res/cardsfolder/upcoming/arachnoid_adaptation.txt

```diff
@@ -0,0 +1,6 @@
+Name:Arachnoid Adaptation
+ManaCost:G
+Types:Instant
+A:SP$ Pump | Cost$ G | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +2 | NumDef$ +2 | KW$ Reach | SubAbility$ DBUntap | SpellDescription$ Target creature gets +2/+2 and gains reach until end of turn. Untap it.

```

Comment:
Cost isn't needed (anymore) unless it has some sort of alt-cost or additional cost